docs/design/client-sessions.md

```diff
@@ -4,18 +4,18 @@ sidebar_position: 4
 
 # Client Sessions
 
-A _client session_ is a sequence of alternating [requests](./client-requests.md) and replies between a
-client and a cluster.
+A _client session_ is a series [requests](./client-requests.md) and replies sent between a client
```

Comment:
:thinking: I think "sequence" is more correct here? At least in my head it has a stronger denotation of order.